app/src/views/search-engine/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from 'react';
+import { useQuery } from '@apollo/client';
+import { useDebounce } from 'use-lodash-debounce';
+import { Container } from '@material-ui/core';
+import SearchBar from './components/search-bar';
+import ArticlesList from './components/articles-list';
+import { GET_ARTICLES } from './graphql/queries';
+
+const SearchEngine = () => {
```

Comment:
Seeing how this is the main component of the application some documentation would be good

Reply:
+1

Reply:
What do you think is relevant information to include into the component?